src/AWS.Deploy.Common/DeploymentManifest/DeploymentManifestEngine.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using AWS.Deploy.Common.IO;
+using Newtonsoft.Json;
+
+namespace AWS.Deploy.Common.DeploymentManifest
+{
+    /// <summary>
+    /// This class contains the helper methods to update the deployment manifest file
+    /// that keeps track of the save CDK deployment projects.
+    /// </summary>
+    public class DeploymentManifestEngine
+    {
+        private readonly IDirectoryManager _directoryManager;
+        private readonly IFileManager _fileManager;
+        private readonly string _targetApplicationFullPath;
+        private readonly string _saveCdkDirectoryFullPath;
+
+        public DeploymentManifestEngine(string targetApplicationFullPath, string saveCdkDirectoryFullPath)
```

Comment:
I expect both `targetApplicationFullPath` and `saveCdkDirectoryFullPath` as parameters to public methods. Also add DeploymentManifestEngine to dependencies [here](https://github.com/aws/aws-dotnet-deploy/blob/main/src/AWS.Deploy.CLI/Extensions/CustomServiceCollectionExtension.cs#L40) and use constructor DI to obtain an instance of `IDeploymentManifestEngine`.

Reply:
Fixed